gulpfile.js/extras/craft/gulprev/twigextensions/GulpRevTwigExtension.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Craft;
+
+use Twig_Extension;
+use Twig_Filter_Method;
+
+class GulpRevTwigExtension extends Twig_Extension
+{
+    public function getName()
+    {
+        return 'gulpRev';
+    }
+
+    public function getFilters()
```

Comment:
I think I'd prefer functions, so we can keep implementation across platforms more consistent. Can you or @mzlock update this and match the methods and naming conventions used in the rails helper? 
```rb
gulp_asset_path('image/asset.jpg') # -> /image/logo-n39o4orb81.png
gulp_js_path('app.js')                      # -> /javascripts/app-f43e9abc11.js
gulp_css_path('app.css')                # -> /stylesheets/app-d29e4cdb76.css
gulp_img_path('logo.png')              # -> /images/logo-n39o4orb81.png
```

Reference: https://github.com/vigetlabs/gulp-starter/pull/223/files#diff-2985f81dc18e56ce0295c3580ae70f13R2